### ISSUE TITLE
Missing Items Day 3 

### ISSUE THREAD

**Issue by jlp20002** [jeanropke/RDR2CollectorsMap#47]
1. 1797 Gold Eagle: To the right of between the R and O in Roanoke

2. Ursula Citrine Earrings: one to the left of the shack South of the railroad tracks and one at the big house south of Van Horn

3. Magnate Turquoise Ring: left of the border of Ambarino and Roanoke Ridge

4. Beauchēne Ruby Earrings: one by O'Creagh Run Shack and one at the top of the River bend by Annesburg

5. Feldspar Arrowhead: one under second N in Annesburg one near gazebo by pier at Braithwaite Manor

6. 1792 Quarter: Under Shootout Series near Annesburg

7. Chipped Arrowhead: Above R in Annesburg

8. Quartz Arrowhead: Above R in new Hanover and northeast of Elysian pool

9. 1789 Penny: one north of Pelle Pearl Necklace near Van Horn above second road one above Mattock Pond

10. Crude Arrowhead: Near Gravestone northeast of elimination series by Bluewater Marsh

11. Sterling Tooth Earrings: By elimination series by Bluewater Marsh

12. Ebony Hairpin: Southeast of house at the H in Bluewater Marsh

13. Aubrey Onyx Ring: Above U in Bluewater Marsh next to shack

14. Raw Arrowhead: Bottom Right of B in Bluewater Marsh

15. Cardinal Flower: One at the bottom right of B in Bluewater Marsh and one southeast of endicott diamond earrings near Caliga Hall 

16. Dane Topaz Necklace: South of Aubrey Onyx Ring in Blackwater

17. Rough Arrowhead: East of Tennessee Whisky Under Braithwaite Manor

18. Endicott Diamond Earrings: island Southeast of the words Kamassa River near Caliga Hall

20. 1792 Nickel: North of R in Rhodes above the road

21. Flint Arrowhead: South of the 2nd t in Southfield Flats

**Comment by iligalodin** [comment]
> 11. Sterling Tooth Earrings: By elimination series by Bluewater Marsh

cant find anything close to the elimination series, could you provide a screenshot on the map?

**Comment by iligalodin** [comment]
are you sure this is day three? i've looked at multiple locations but cant find none

**Comment by jlp20002** [comment]
Unfortunately, I didn't get a screenshot of that one. I definitely found them after the day cycled to day three, but I was playing before the days switched... These items may be from the other day's set.

**Comment by iligalodin** [comment]
will check once the cycle hits day 2 again

**Comment by Jugrnot8** [comment]
I verified the ones I could of his that are in Scarlet Meadows. It looks like he has a mix of stuff from day 2 and day 3 in his list. 

5. I found a different item within the same collection 
9. I found a different item within the same collection 
15. No findings. Description is very vague.
17. I found a different item within the same collection
18.I found a different item within the same collection 
19. I did get the same item as he did on Day 3
20. I found a different item within the same collection
